components/Web3Provider.tsx

```diff
@@ -226,6 +259,8 @@ export const Web3Provider = ({ children }: Web3ProviderProps): JSX.Element => {
 				handleDisconnectWallet,
 				currentUser,
 				updateCurrentUser,
+				createIdentity,
+				calculateUserReputation,
```

Comment:
This looks okay, but I haven't tested these yet or played with Interep enough to know exactly what it does for us. Looking at these changes though, since `createIdentity` and `calculateUserRepuation` aren't used outside of this context provider, they don't really need to be exported out as values until they're needed externally or in other components part of other functions. I can see that it's all baked into the connect wallet, which may be what we could start out with.

So I think it's a good start but we can remove `L262-263` and also, I'd like to check on their docs and understand what we're actually doing here before mergin this into the codebase.